server/plugin.go

```diff
@@ -167,17 +164,23 @@ func (p *Plugin) MessageWillBePosted(c *plugin.Context, post *model.Post) (*mode
 	}
 
 	msg := post.Message
+	replacements := p.getPermalinkReplacements(msg)
+	if len(replacements) == 0 {
+		return nil, ""
+	}
 	info, err := p.getGitlabUserInfoByMattermostID(post.UserId)
+	if err != nil && err.ID == APIErrorIDNotConnected {
+		p.API.LogDebug("Processing permalink of the post", err.Message)
+	}
 	if err != nil {
-		p.API.LogError("error in getting user info", "error", err.Message)
+		p.API.LogDebug("Error in getting user info", "error", err.Message)
```

Comment:
why these changes

Reply:
All LogError's in this PR is changed to LogDebug due to the passive nature of this feature